instances/latest/contentTypes/application_vnd.nest-desktop.model.v4+json.jsonld

```diff
@@ -0,0 +1,19 @@
+{
+    "@context": {
+      "@vocab": "https://openminds.om-i.org/props/"
+    },
+    "@id": "https://openminds.om-i.org/instances/contentTypes/application_vnd.nest-desktop.model.v4+json",
+    "@type": "https://openminds.om-i.org/types/ContentType",
+    "dataType": null,
```

Comment:
Associative array then?

Reply:
I mean, yes, because it's a JSON document, but semantically it's a configuration.

Reply:
there can be more than one data type. let's now not integrate this, but I'd like this field being filled for all content types in the long run